speech.go

```diff
@@ -33,6 +33,8 @@ const (
 	SpeechResponseFormatOpus SpeechResponseFormat = "opus"
 	SpeechResponseFormatAac  SpeechResponseFormat = "aac"
 	SpeechResponseFormatFlac SpeechResponseFormat = "flac"
+	SpeechResponseFormatWav SpeechResponseFormat = "wav"
+	SpeechResponseFormatPcm SpeechResponseFormat = "pcm"
```

Comment:
@LinuxSploit thank you for the PR! Please run goimports so the file would get accepted by the linter!